client/js/libs/handlebars/ircmessageparser/merge.js

```diff
@@ -0,0 +1,34 @@
+"use strict";
+
+const _ = require("lodash");
```

Comment:
I thought we were using it there already but I was wrong, my bad. I'll remove, thanks for catching this!